xinference/deploy/cmdline.py

```diff
@@ -925,6 +925,18 @@ async def chat_internal():
             )
 
 
+@cli.command("vllm-models", help="Query and display models compatible with VLLM.")
+@click.option("--endpoint", "-e", type=str, help="Xinference endpoint.")
+def vllm_models(endpoint: Optional[str]):
+    endpoint = get_endpoint(endpoint)
+    client = RESTfulClient(base_url=endpoint)
+    client._set_token(get_stored_token(endpoint, client))
+    vllm_models_list = client.vllm_models()
+    print("VLLM Models:")
+    for model in vllm_models_list:
+        print(model)
```

Comment:
Couly you please show the chat and generate models supported by vllm separately?